node/src/api/network.rs

```diff
@@ -0,0 +1,81 @@
+mod handlers;
+mod requests;
+mod responses;
+
+use crate::api::dto::{Cursor, HexId};
+use warp::Filter;
+use crate::bc::BlockchainRef;
+use std::convert::Infallible;
+use crate::api::warp_utils::{handle2, handle1};
+
+pub fn routes(bc: BlockchainRef) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+    status(bc.clone())
+        .or(mempool(bc.clone()))
+        .or(blocks(bc.clone()))
+        .or(block(bc.clone()))
+        .or(tx(bc.clone()))
+        .or(submit(bc))
+}
+
+fn status(bc: BlockchainRef) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
```

Comment:
Would it be more readable if those routes were declared as local variables instead of functions? Then we wouldn't need those long return type definitions all over the place.

Reply:
Updated by https://github.com/stellar/slingshot/pull/492/commits/e4982808103defc2922d75673fb364564c9ad6a9